pkg/xds/server/v3/reconcile.go

```diff
@@ -172,17 +170,6 @@ func (s *TemplateSnapshotGenerator) GenerateSnapshot(ctx context.Context, xdsCtx
 		reconcileLog.Error(err, "failed to generate a snapshot", "proxy", proxy, "template", template)
 		return nil, err
 	}
-	allPolicies := plugins.Plugins().PolicyPlugins()
-	for _, policyName := range policies.Policies {
-		policy, exists := allPolicies[policyName]
-		if !exists {
-			reconcileLog.Error(errors.Errorf("policy doesn't exist"), "failed to apply policy's changes", "policyName", policyName)
-			continue
-		}
-		if err := policy.Apply(rs, xdsCtx, proxy); err != nil {
-			return nil, errors.Wrapf(err, "could not apply policy plugin %s", policyName)
-		}
-	}
 	for _, hook := range s.ResourceSetHooks {
```

Comment:
Is this likely to cause a problem with resourceSetHooks running before policy Plugins?

Reply:
They won't run before policy plugins, they should be run in the `gen.Generate` step above this line.